public/app/features/alerting/unified/utils/misc.ts

```diff
@@ -206,7 +206,9 @@ const alertStateSortScore = {
   [PromAlertingRuleState.Firing]: 1,
   [GrafanaAlertState.Error]: 1,
   [GrafanaAlertState.Pending]: 2,
+  [GrafanaAlertState.Recovering]: 2,
   [PromAlertingRuleState.Pending]: 2,
+  [PromAlertingRuleState.Recovering]: 2,
```

Comment:
I wonder if we should assign 2 to `Recovering` and add +1 to everything that currently has 2 or more

Reply:
But then , we would have `Recovering` with higher score than `Pending` 🤔

Reply:
The sort score is going to be irrelevant soon anyway so I'm not partial to either :D let's keep it as it is in the PR for now